src/blocks/gallery-carousel/index.js

```diff
@@ -39,6 +39,17 @@ const settings = {
 	supports: {
 		align: [ 'wide', 'full' ],
 	},
+	example: {
+		attributes: {
+			gridSize: 'lrg',
+			gutter: 5,
+			images: [
+				{ url: '/wp-content/plugins/coblocks/dist/images/examples/gallery-1.jpg' },
```

Comment:
One  bit I'm not 100% about is how I'm pulling images from the plugin to use in the examples for the gallery blocks. Thoughts?

Reply:
I think this is perfectly fine. We're not adding too many images and the file sizes are not large at all. I would prefer loading local images over potentially breaking the experience if a remote image fails to load.